.storybook/stories/LendHomepageLoanCard.stories.js

```diff
@@ -85,6 +85,12 @@ export const Default = () => ({
 				:loan="loan"
 				:percent-raised="percentRaised"
 			/>
+			<component is="style">
```

Comment:
This is a hack that basically inserts a style tag and allows for styling without the need to add a :style attribute to the component in the story

Reply:
oh weird! but neat :)

I think the part I don't like here is that it's opaque where .lend-homepage-loan-card class resides. I'm guessing it's the first node inside lend-homepage-loan-card component. One idea is to add a class in this story on the component, maybe `<lend-homepage-loan-card class="loan-card" ...>` and then inside your style tag `.loan-card { ... }`. This would tell the whole story of how you need to have this CSS applied to outer node of the component to get the border radius/box shadow effect.

Reply:
@ryan-ludwig good suggestion, updated